update/genesis/import.go

```diff
@@ -307,7 +341,11 @@ func (si *stateImport) importState(fileName string, trieKey string) error {
 	}
 
 	oldRootHash := value
-	log.Debug("old root hash", "value", oldRootHash)
+	log.Debug("old root hash", "value", oldRootHash, "read root hash", rootHash)
```

Comment:
log.Debug("root hash", "old", oldRootHash, "read", rootHash)

Reply:
not needed. deleted.